src/Utility/Utils.cs

```diff
@@ -1,102 +1,131 @@
-                        {
-                            // Stream is expected, so convert the byte array to a MemoryStream.
-                            value = new MemoryStream(blobBytes);
-                        }
-                    }
-
-                    // When the input is 'String', we don't attempt to convert it to bytes because if the blob
-                    // is in fact a binary file, it's impossible to get the right bytes back even if we use the
-                    // same encoding used by the host when converting those bytes to string.
-
-                    break;
-                default:
-                    break;
-            }
-
-            return value;
-        }
-
-        /// <summary>
-        /// Helper method to do additional transformation on the output value.
-        /// </summary>
-        internal static object TransformOutBindingValueAsNeeded(string bindingName, ReadOnlyBindingInfo bindingInfo, object value)
-        {
-            switch (bindingInfo.Type)
-            {
-                case "http":
-                    // Try converting the value to HttpResponseContext if it's not already an object of such type.
-                    if (value is HttpResponseContext)
-                    {
-                        break;
-                    }
-
-                    try
-                    {
-                        value = LanguagePrimitives.ConvertTo<HttpResponseContext>(value);
-                    }
-                    catch (PSInvalidCastException ex)
-                    {
-                        string errorMsg = string.Format(PowerShellWorkerStrings.FailToConvertToHttpResponseContext, bindingName, ex.Message);
-                        throw new InvalidOperationException(errorMsg);
-                    }
-
-                    break;
-                default:
-                    break;
-            }
-
-            return value;
-        }
-    }
-}
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using System.Text;
+using Microsoft.PowerShell.Commands;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Utility
+{
+    internal class Utils
+    {
+        internal static CmdletInfo ImportModuleCmdletInfo = new CmdletInfo("Import-Module", typeof(ImportModuleCommand));
+        internal static CmdletInfo RemoveModuleCmdletInfo = new CmdletInfo("Remove-Module", typeof(RemoveModuleCommand));
+        internal static CmdletInfo GetJobCmdletInfo = new CmdletInfo("Get-Job", typeof(GetJobCommand));
+        internal static CmdletInfo RemoveJobCmdletInfo = new CmdletInfo("Remove-Job", typeof(RemoveJobCommand));
+
+        private static InitialSessionState s_initialSessionState = null;
+
+        internal static InitialSessionState GetInitialSessionState()
+        {
```

Comment:
This can be changed to Lazy and expose InitialSessionState as property and make GetInitialSessionState() as private to just create an instance of InitialSessionState like below:
 private static Lazy<InitialSessionState> s_initialSessionState = new Lazy<InitialSessionState>(GetInitialSessionState);
        internal static InitialSessionState InitialSessionStateInstance => s_initialSessionState.Value;

        private static InitialSessionState GetInitialSessionState()
        {
            var iss = InitialSessionState.CreateDefault();
            iss.ThreadOptions = PSThreadOptions.UseCurrentThread;
            iss.EnvironmentVariables.Add(
                new SessionStateVariableEntry(
                    "PSModulePath",
                    FunctionLoader.FunctionModulePath,
                    description: null));

            // Setting the execution policy on macOS and Linux throws an exception so only update it on Windows
            if (Platform.IsWindows)
            {
                // This sets the execution policy on Windows to Unrestricted which is required to run the user's function scripts on
                // Windows client versions. This is needed if a user is testing their function locally with the func CLI.
                iss.ExecutionPolicy = Microsoft.PowerShell.ExecutionPolicy.Unrestricted;
            }

            return iss;
        }

Reply:
Sure, I can make it a `Lazy<T>` property.